elm/wizard.py

```diff
@@ -603,12 +619,62 @@ def query_vector_db(self, query, probes=25, limit=100):
 
         return strings, scores, best
 
+    def _format_refs(self, refs, ids):
```

Comment:
This method does quite a bit of new work - it might be worth adding one or more tests dedicated to it's functionality. 

For example, something like this to start (should go in `test_wizard_postgres.py`; I am eyeballing this code, so it may not work without further modifications:

```
def test_format_refs():
    wizard = EnergyWizardPostgres(db_host='Dummy', db_port='Dummy',
                                  db_name='Dummy', db_schema='Dummy',
                                  db_table='Dummy',
                                  boto_client=BotoClient(),
                                  meta_columns=['title', 'url', 'id'])
    refs = [{'title': chr(34), 'url': "test.com", "id": 5}, 
            {'title': 5, 'url': "test_2.com", "id": 7}, 
            {'title': "three", 'url': "test_3.com", "id": 3, "extra_col": 0}, 
            {'url': "test_3.com", 'title': "three", "id": 3},
            {'title': "four", 'url': "test_4.com", "id": 9}]
    ids = np.array([7, 3, 5])
    
    expected_out = [{'title': "5", 'url': "test_2.com", "id": 7},  
                    {'title': "three", 'url': "test_3.com", "id": 3}, 
                    {'title': "", 'url': "test.com", "id": 5}]
    expected_out = [json.dumps(ref) for ref in expected_out]
    
    assert wizard._format_refs(refs, ids) == expected_out 
```

It's kind of a big test, so you could break it out into a few cases - that's probably a better way to go anyways and good practice for writing tests!